pkg/network/public/public.go

```diff
@@ -513,6 +515,15 @@ func publicConfig(iface *pkg.PublicConfig) (ips []*net.IPNet, routes []*netlink.
 	return ips, routes, nil
 }
 
+func ensurePublicResolve() error {
+	path := filepath.Join("/etc", "netns", PublicNamespace)
+	if err := os.MkdirAll(path, 0755); err != nil {
+		return errors.Wrap(err, "failed to create public netns directory")
+	}
+	path = filepath.Join(path, "resolv.conf")
+	return ioutil.WriteFile(path, []byte(defaultPublicResolveConf), 0544)
```

Comment:
ioutil package is deprecated since go 1.16 functionality is now available in `io` and `os` packages

Reply:
I think adding the ipv6 nameserver anyway won't hurt even if there is no ipv6